src/solvers/z3/z3_conv.cpp

```diff
@@ -16,6 +16,8 @@ smt_convt *create_new_z3_solver(
   array_iface **array_api,
   fp_convt **fp_api)
 {
+  if(options.get_bool_option("z3-debug"))
+    Z3_open_log("z3.log");
```

Comment:
Shall we allow the user to provide the filename for the log?

Reply:
done